### PR TITLE
added types to npm package

### PR DESCRIPTION
**Pull request by pavletto**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,13 +73,17 @@
...
     "3D",
     "webgl",
     "globe",
     "og"
-  ]
-}
\ No newline at end of file
+  ],
+  "dependencies": {
+    "@openglobus/og": "^0.10.6"
```
Comment (tbo47) [inline package.json]: Why adding a dependency to `"@openglobus/og` here?

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -16,7 +16,6 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12
-      - run: npm install
       - run: npm ci
```
Comment (tbo47) [inline .github/workflows/npm-publish.yml:19]: `npm install` generates a `package-lock.json`. Then `npm ci` uses the `package-lock.json`.
Removing `npm install` is a good idea but we need to add the `package-lock.json` to the repo. (and remove `package-lock.json` from `.gitignore:20`)

File: package.json (hunk 1 above)
Comment (pavletto) [reply package.json]: Randomness - looked at the contents of our package. Thanks!

File: .github/workflows/npm-publish.yml (hunk 2 above)
Comment (pavletto) [reply .github/workflows/npm-publish.yml:19]: Agree

Comment (tbo47) [reply .github/workflows/npm-publish.yml:19]: I see this in package-lock.json
```
      "dependencies": {
        "@openglobus/og": "^0.10.6"
      },
```
Did you generate it with the latest package.json?

**Review by Zemledelec — APPROVED** [review]

**Pull request merged**